load_test/xmpp_amqp.erl

```diff
@@ -242,11 +242,19 @@ xmpp_do(#{neighborIDs := NeighborIDs,
             send_presence_available(Client);
         {scheduled, send_message} ->
             schedule(send_message, SendMessageAfter),
-            send_messages_to_neighbors(Client, NeighborIDs)
+            send_messages_to_neighbors(Client, NeighborIDs);
+        {system, {From, Mref}, get_state} -> % allows to do sys:get_state(Pid)
```

Comment:
Why do we need this?

Reply:
I found it pretty useful for debugging purpose.

Is possible to do:
```
[{_, P, _, _} | _] = supervisor:which_children(amoc_users_sup),
sys:get_state(P)
```

Of course we can obtain such info with tracing, however I find this solutions more suitable for some cases. You just do not have to wait for some event, you just ask a worker for its state.